README.md

```diff
@@ -50,26 +50,25 @@ This package is for those interested in digital pathology: including graduate st
 
 **`tiatoolbox`** is a multipurpose name that we use for 1) a certain computer program, 2) a Python package of related programs, created by us at the TIA Centre to help people get started in Digital Pathology, 3) this repository, 4) a certain virtual environment.
 
-
 ### Developers
 
 Anyone wanting to contribute to this repository, please first look at our [Wiki](https://github.com/TissueImageAnalytics/tiatoolbox/wiki) and at our web page for [contributors](https://github.com/TissueImageAnalytics/tiatoolbox/blob/master/CONTRIBUTING.rst). See also the *Prepare for development* section of this document.
 
 ### Links, if needed
+
 The [bash](https://www.gnu.org/software/bash) shell is available on all commonly encountered platforms. Commands in this README are in bash. Windows users can use the command prompt to install conda and python packages.
 
 [conda](https://github.com/conda/conda) is a management system for software packages and [virtual environments](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html). To get `conda`, download [Anaconda](https://www.anaconda.com/), which includes hundreds of the most useful Python packages, using 2GB disk space. Alternatively, [miniconda](https://docs.conda.io/en/latest/miniconda.html) uses 400MB, and packages can be added as needed.
 
 [Github](https://github.com/about) is powered by the version control system [git](https://git-scm.com/), which has many users and uses. In Github, it is used to track versions of code and other documents.
 
-
 ### Examples Taster
 
 1. [Click here](https://github.com/TissueImageAnalytics/tiatoolbox/tree/develop/examples) for [jupyter notebooks](https://jupyter.org/), hosted on the web, with demos of `tiatoolbox`. All necessary resources to run the notebooks are remotely provided, so you don't need to have Python installed on your computer.
-2. Click on a filename with suffix `.ipynb` and the notebook will open in your browser.
-3. Click on one of the two blue checkboxes in your browser window labelled either **Open in Colab** or **Open in Kaggle**: [colab](https://colab.research.google.com/notebooks/intro.ipynb#) and [kaggle](https://www.kaggle.com/) are websites providing free-of-charge platforms for running jupyter notebooks.
-4. Operate the notebook in your browser, editing, inserting or deleting cells as desired.
-5. Changes you make to the notebook will last no longer than your colab or kaggle session.
+1. Click on a filename with suffix `.ipynb` and the notebook will open in your browser.
```

Comment:
It has rendered correctly but I am not sure if this should be `1.`

Reply:
See https://mdformat.readthedocs.io/en/stable/users/style.html#, the 1. numbering is used to reduce diff size when making edits. The number before the period is NOT used when determining the number when the document is rendered. This is purely done from ordering. Hence, using 1. for all items reduces edit diff size when changing lists.